test/contrib/rails/rack_middleware_test.rb

```diff
@@ -58,6 +58,7 @@ class FullStackTest < ActionDispatch::IntegrationTest
 
     assert_equal(render_span.name, 'rails.render_template')
     assert_equal(render_span.span_type, 'template')
+    assert_equal(render_span.service, Datadog.configuration[:rails][:service_name])
```

Comment:
An extra test that seems to have been overlooked: I unintentionally broken this `render_span`'s service tag while making minor changes to the template rendering integration and no test failed.
I added this test, as it would have caught my mistake in first place, and it should provide a small safeguard for the future.

Reply:
:+1: